server/database/models/UserRepository.js

```diff
@@ -13,10 +13,10 @@ class UserRepository extends AbstractRepository {
   async create(user) {
     const [result] = await this.database.query(
       `
-      INSERT INTO ${this.table} (email, password)
+      INSERT INTO ${this.table} (email, hashed_password)
```

Comment:
Is it recomanded to put hashed_password instead of password ? (Marco, Matthieu ? 👀)

Reply:
hashed_password 👍

Reply:
Than you !